display/projectiles.py

```diff
@@ -29,10 +35,13 @@ def __init__(self, icon, velocity, oscillate_height = 0, oscillate_rate = 1):
 
     def draw_image(self):
         ifile = pygame.image.load(os.path.join("assets", "items", self.icon + ".png"))
-        self.image = pygame.transform.scale(ifile, (150, 150))
+        self.image = pygame.transform.scale(ifile, (15*Projectile.pixel_scale, 15*Projectile.pixel_scale))
```

Comment:
Please could you clarify where `15` comes from here?

Reply:
Ye the size of the image yo.